lib/hypertrack.dart

```diff
@@ -26,23 +28,31 @@ class HyperTrack {
   /// provided. In case of frequent changes, it is possible, that intermediate
   /// states could be lost, so if more real-time responsiveness is required
   /// it is recommended to use [addGeotag] for passing that data.
-  void setDeviceName(String deviceName) async =>
-      _pluginInterface.setDeviceName(deviceName);
+  Future<void> setDeviceName(String deviceName) async =>
+      _sdkWrapper.setDeviceName(deviceName);
 
   /// Returns `true` if tracking was started.
   ///
   /// This doesn't actually means that SDK collecting location data, but only
   /// allows you to check the current state of tracking state switch.
   /// For details on whether tracking actually happens or not check [onTrackingStateChanged] events.
-  Future<bool> isRunning() async {
-    return await _pluginInterface.isRunning();
-  }
+  Future<bool> isRunning() =>
+      _sdkWrapper.isRunning();
+
+  Future<bool> isTracking() =>
+      _sdkWrapper.isTracking();
+
+  Future<Availability> getAvailability() =>
```

Comment:
why not `isAvailable` boolean?

Reply:
It's because this method returns Future of enum `Available`. 

As long as we don't simplify this enum I would consider this naming `getAvailability` as correct.